common/lib/calc/calc/tests/test_calc.py

```diff
@@ -190,12 +190,12 @@ def test_trig_functions(self):
 
         # Include those where the real part is between -pi/2 and pi/2
         arcsin_inputs = ['-0.707', '0', '0.5', '0.588', '1.298 + 0.635*j', '-1.1', '1.1']
-        arcsin_angles = [-0.785, 0, 0.524, 0.629, 1 + 1j, -1.570 + 0.443j, 1.570 - 0.443j]
+        arcsin_angles = [-0.785, 0, 0.524, 0.629, 1 + 1j, -1.570 + 0.443j, 1.570 + 0.443j]
```

Comment:
I’m a little curious why the two changes in this file were required?

Reply:
The value "1.1" is outside the valid range of inputs for `arcsin` and `arccos`, but this test case had been added to make sure that a number was returned rather than `NaN` for assorted reasons.  The NumPy upgraded flipped the sign of the second component in the output, but a quick discussion with the people who added the test revealed that this doesn't really matter; the output is kind of meaningless anyway, it just has to be a number.